### ISSUE TITLE
Erroneous 'Redefing built-in' Warning

### ISSUE THREAD

**Issue by 1John419** [PyCQA/pylint#4027]
## Environment data

-   VS Code version: 1.52.1
-   pylint version: 2.6.0
-   OS and version: Windows 10 20H2 (OS Build 19042.685)
-   Python version: v3.9.0:9cf6752

```
from sys import exit
exit()

```

## Expected behaviour

No errors or warnings

## Actual behaviour

{
	"resource": "/d:/Users/Clayton/Code/Python/JK/bug.py",
	"owner": "python",
	"code": "redefined-builtin",
	"severity": 4,
	"message": "Redefining built-in 'exit'",
	"source": "pylint",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 1,
	"endColumn": 1
}

**Comment by hippo91** [comment]
@1John419 thanks for your report. However i think that `pylint` is right here because importing the `exit` function from the `sys` module indeed masks the builtin `exit` function.
You can overcome this warning by doing this:
```
import sys
sys.exit()
```
